src/components/Documentation/Markdown/index.tsx

```diff
@@ -50,11 +50,62 @@ const Abbr: React.FC<{ children: [string] }> = ({ children }) => {
   return <Tooltip text={children[0]} />
 }
 
+const Cards: React.FC<{
+  children: Array<object | string>
+}> = ({ children }) => {
+  return (
+    <div className={styles.cards}>
+      {Array.isArray(children)
```

Comment:
I'm still not sure about this - if we have this check, we should have a different type above to my mind ... it is strange that `ts-lint` complains - any idea why is it happening?